docs/user_guides/fs/storage_connector/creation/jdbc.md

```diff
@@ -36,16 +36,26 @@ Head to the Storage Connector View on Hopsworks (1) and set up a new storage con
 
 Enter the details for your JDBC enabled database.
 
+<figure markdown>
+  ![JDBC Connector Creation](../../../../assets/images/guides/fs/storage_connector/jdbc_creation.png)
+  <figcaption>JDBC Connector Creation Form</figcaption>
+</figure>
```

Comment:
maybe we could just suffice with 1 image?

Reply:
fixed